apps/package.json

```diff
@@ -67,6 +67,7 @@
     "@types/mocha": "^10.0.1",
     "@types/react-bootstrap": "^0.32.36",
     "@types/react-router-dom": "^5.3.3",
+    "@types/react-select": "^1.2.1",
```

Comment:
Are the updates to this and `yarn.lock` part of this PR?

Reply:
yeah, the old EnrollForm component is still using `react-select` for the dropdown, so converting the component to typescript meant needing to add the types for `react-select`. I'm guessing we'll eventually swap in DSCO components and won't need these anymore

Reply:
Gotcha just wanted to check that it wasn't a merge/branching leftover change or something!